packages/nyxt-deb/nyxt-deb.pacscript

```diff
@@ -1,6 +1,7 @@
 name="nyxt-deb"
 version="2.2.4"
 gives="nyxt"
+repology="nyxt"
```

Comment:
I guess the AUR (currently has the latest version of the package) tracks pre-releases too, so I guess we would have to track some other repo.

Select a repo from here: https://repology.org/project/nyxt/versions

Reply:
So what should I do? version [3](https://github.com/atlas-engineer/nyxt/releases/tag/3) was a test so after that should be fine. If you want I can temporarily remove `repology`.

Reply:
Which repository do you want to track?

Reply:
```suggestion
repology=("project: nyxt" "repo: void_x86_64")
```

I guess if you're unable to decide Void Linux's repository would be a good choice.

Reply:
Never knew I could track a specific repo